pyccel/parser/semantic.py

```diff
@@ -256,6 +256,7 @@ def annotate(self, **settings):
 
     def _get_new_name(self, current_name):
         if current_name not in self._possible_names:
+            self._possible_names.add(current_name)
```

Comment:
Is `self._possible_names` a list? Should't we use `append` instead of `add`?

Reply:
It's a set

Reply:
I see, that makes sense